homeassistant/components/home_connect/strings.json

```diff
@@ -17,6 +17,169 @@
       "default": "[%key:common::config_flow::create_entry::authenticated%]"
     }
   },
+  "entity": {
+    "select": {
+      "program": {
+        "name": "Program",
+        "state": {
+          "consumer_products-cleaning_robot-program-cleaning-clean_all": "Clean all",
+          "consumer_products-cleaning_robot-program-cleaning-clean_map": "Clean map",
+          "consumer_products-cleaning_robot-program-basic-go_home": "Go home",
+          "consumer_products-coffee_maker-program-beverage-ristretto": "Ristretto",
+          "consumer_products-coffee_maker-program-beverage-espresso": "Espresso",
+          "consumer_products-coffee_maker-program-beverage-espresso_doppio": "Espresso doppio",
+          "consumer_products-coffee_maker-program-beverage-coffee": "Coffee",
+          "consumer_products-coffee_maker-program-beverage-x_l_coffee": "XL coffee",
+          "consumer_products-coffee_maker-program-beverage-caffe_grande": "Caffe grande",
+          "consumer_products-coffee_maker-program-beverage-espresso_macchiato": "Espresso macchiato",
+          "consumer_products-coffee_maker-program-beverage-cappuccino": "Cappuccino",
+          "consumer_products-coffee_maker-program-beverage-latte_macchiato": "Latte macchiato",
+          "consumer_products-coffee_maker-program-beverage-caffe_latte": "Caffe latte",
+          "consumer_products-coffee_maker-program-beverage-milk_froth": "Milk froth",
+          "consumer_products-coffee_maker-program-beverage-warm_milk": "Warm milk",
+          "consumer_products-coffee_maker-program-coffee_world-kleiner_brauner": "Kleiner brauner",
+          "consumer_products-coffee_maker-program-coffee_world-grosser_brauner": "Grosser brauner",
+          "consumer_products-coffee_maker-program-coffee_world-verlaengerter": "Verlaengerter",
+          "consumer_products-coffee_maker-program-coffee_world-verlaengerter_braun": "Verlaengerter braun",
+          "consumer_products-coffee_maker-program-coffee_world-wiener_melange": "Wiener melange",
+          "consumer_products-coffee_maker-program-coffee_world-flat_white": "Flat white",
+          "consumer_products-coffee_maker-program-coffee_world-cortado": "Cortado",
```

Comment:
It is true that can look very tedious at templating, but they almost look like the API docs specification (because of the transformation I perform) and every possible value documented is there.
Splitting them will be pretty helpful for templating and other UX cases, but we will need to find a logic that is able to include all the possible states. Here I think we could split up to `program` (without including it).

PD. `-` is supported, I cannot find where is the regex for `strings.json` keys, but it allowed it

Reply:
I think I've reached a halfway solution.
Instead of doing those translations and having the bsh keys at `string.json` , I created a dictionary that contains all the possible options and its translation keys.
I did the same on the other PR with this issue